### PR TITLE
fix: correctly parse multistage dockerfiles written in lowercase

### PR DESCRIPTION
**Pull request by RotemS**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Bumping snyk-docker-plugin (contains a fix for parsing dockerfiles written in lowercase)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "proxy-from-env": "^1.0.0",
     "semver": "^6.0.0",
     "snyk-config": "^2.2.1",
-    "snyk-docker-plugin": "1.33.1",
+    "snyk-docker-plugin": "1.37.3",
```
Comment (dkontorovskyy) [inline package.json]: There is already `1.38.0`, maybe upgrade to that version?

Comment (RotemS) [reply package.json]: You're right, I missed it 🙈 Changed

**Comment by lili2311** [comment]
@RotemS please change the commit to `feat` this is a change that release new features, `chore` will not trigger a release. Please could you also update the commit message to explain what will be included in this new release? The commit is part of release notes, currently it doesn't explain

**Comment by RotemS** [comment]
> @RotemS please change the commit to `feat` this is a change that release new features, `chore` will not trigger a release. Please could you also update the commit message to explain what will be included in this new release? The commit is part of release notes, currently it doesn't explain

@lili2311 hope it's ok, changed to `fix` as I thought it was more suitable. Also updated the commit message.

**Pull request merged**